terracumber_config/tf_files/SUSEManager-Head-NUE.tf

```diff
@@ -124,20 +124,22 @@ module "cucumber_testsuite" {
       }
     }
     server = {
+      image = "sles15sp4o"
       provider_settings = {
         mac = "aa:b2:93:01:00:b1"
         memory = 10240
       }
     }
     proxy = {
+      image = "sles15sp4o"
```

Comment:
In theory, that's the default value, we don't need to specify it.

Reply:
https://github.com/uyuni-project/sumaform/blob/master/modules/proxy/main.tf#L12

Reply:
I thought `sles15sp3o` is the default?

Reply:
Fixed.